extensions/typescript-language-features/src/languageFeatures/tsconfig.ts

```diff
@@ -122,5 +175,15 @@ export function register() {
 		languages.map(language =>
 			patterns.map((pattern): vscode.DocumentFilter => ({ language, pattern }))));
 
-	return vscode.languages.registerDocumentLinkProvider(selector, new TsconfigLinkProvider());
+	return vscode.Disposable.from(
+		vscode.commands.registerCommand(OPEN_EXTENDS_LINK_COMMAND_ID, async ({ resourceUri, extendsValue, }: OPEN_EXTENDS_LINK_COMMAND_ARGS) => {
+			const tsconfigPath = await getTsconfigPath(Utils.dirname(resourceUri), extendsValue);
+			if (tsconfigPath === undefined) {
+				void vscode.window.showErrorMessage(localize('openTsconfigExtendsModuleFail', "Failed to resolve {0} as module", extendsValue));
```

Comment:
Added void, as we don't need to wait until user resolve an error.
By adding void we make this line [no-floating-promises](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md) rule aware. I hope it will enabled in https://github.com/microsoft/vscode/issues/128738. 
Or I should throw it instead?